express-notes-api/server.ts

```diff
@@ -0,0 +1,116 @@
+import express from 'express';
+
+import { readFile, writeFile } from 'node:fs/promises';
+
+type Content = {
```

Comment:
This is the data type of a single note and Note below is the data type of the data.json file. I think the names you have chosen are confusing.